project2/style.css

```diff
@@ -0,0 +1,66 @@
+.title {
+    font-family: 'ZCOOL QingKe HuangYou', cursive;
+    color: white;
```

Comment:
This color text on this background color has very low contrast. I recommend using more contrast between your text and background color; your original design had much clearer contrast and was more readable, as a result.